<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>T1WrenchMainWindow</class>
 <widget class="QMainWindow" name="T1WrenchMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Smartisan T1聊天小扳手</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>771</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>聊天专区</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>521</width>
        <height>351</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCellPhoneTextEdit" name="phoneTextEdit"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="weixinQqRadio">
           <property name="text">
            <string>微信、QQ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="replyMailRadio">
           <property name="text">
            <string>回邮件</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="replySmsRadio">
           <property name="text">
            <string>回短信</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="weiboRadio">
           <property name="text">
            <string>微博</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="googlePlusRadio">
           <property name="text">
            <string>Google+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="toClipBoardRadio">
           <property name="text">
            <string>-&gt;剪贴板</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="fromClipBoard">
           <property name="text">
            <string>&lt;-剪贴板</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="qqHintLabel">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>10</y>
        <width>221</width>
        <height>391</height>
       </rect>
      </property>
      <property name="text">
       <string>适用场景示意图</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>521</width>
        <height>84</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QToolButton" name="tbEmoji">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbPicture">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbScreenCapture">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="configurePushButton">
           <property name="text">
            <string>配置</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout"/>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="sendItPushButton">
           <property name="text">
            <string>扳动</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>761</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTextEdit" name="cmdOutputEdit">
       <property name="minimumSize">
        <size>
         <width>580</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>580</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="adbStateLabel">
       <property name="text">
        <string>Disconnected</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="adbStateIndicator">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="T1Wrench.qrc">:/images/red.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCellPhoneTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">qcellphonetextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="T1Wrench.qrc"/>
 </resources>
 <connections/>
</ui>
